### DISCUSSION TITLE
[pixi-spine] IK setting bones with wrong Rotation

### DISCUSSION THREAD

**Discussion by diegomachado** [pixijs/pixijs#8309] [Q&A]
When using simple IKs on my character's feet, they stay pointed up when imported using pixi-spine.
It works perfectly when using spine-ts and also on Unity.

![pixi-spine-ik-issue](https://user-images.githubusercontent.com/104657/165956646-c98bb5e7-7049-4973-8460-e75688707a07.png)

The feet get corrected if I set the IK's Mix parameter to `0` on Spine, but then all animations break because that basically deactivates the IK, and that's why I know this is the root of the issue.

## Environment
Spine: 4.0 Pro
pixi.js: 6.3.0
pixi-spine: 3.0.13
Same behavior on Chrome, Firefox and Brave

**Comment by ivanpopelyshev** [comment]
i have to update to latest official spine runtime first. IK fixes are pushed every 2 months or so :)

**Reply by diegomachado** [reply]
Is it an easy fix? Could I do it myself without previous knowledge on the plugin? 
Just to align my expectations here, I really want to use Pixi on this project. :)

**Reply by ivanpopelyshev** [reply]
My last changes are from september 2021. You have to look all changes to spine-ts official repo and port them here.

Right now i cant spend my time on it, you have to ask in main pixijs repo who's currently in charge of spine plugin.

**Reply by diegomachado** [reply]
Where exactly should I ask @ivanpopelyshev? Also, I should open an Issue, right? 

The pixi-spine repo seems a little abandoned, also, I was going to ask about it on the Discord server but it also seems that it's not official, so I'm kind of lost here.

If you could give me any guidance on what would be the best approach to update the lib by myself I'd be grateful. :)

**Reply by ivanpopelyshev** [reply]
discord is now official, even Mat Groves is there :)

Open issue, put a bounty on update task

**Comment by ivanpopelyshev** [comment]
3.0.14 , should be fixed now :)

There were a few serious chagnes in TransformConstraint and ScaleTimeline
